tendrl/commons/objects/node/atoms/is_node_tendrl_managed/__init__.py

```diff
@@ -9,57 +9,59 @@ def __init__(self, *args, **kwargs):
         super(IsNodeTendrlManaged, self).__init__(*args, **kwargs)
 
     def run(self):
-        node_ids = self.parameters.get('Node[]')
-        if not node_ids or len(node_ids) == 0:
-            raise AtomExecutionFailedError("Node[] cannot be empty")
+        if self.parameters:
```

Comment:
I have handled it in the baseflow, baseatom classes, please remove this change,

more details https://github.com/Tendrl/commons/issues/572